File: 1/Euler1.cs
public class Euler1 {
  public static bool multiple_of(int x, int y) {
    return (x % y) == 0;
  }
  public static void Main() {
    int answer = 0;
    for(int i = 1; i < 1000; i++)
      {
	if(multiple_of(i, 3) || multiple_of(i, 5)) answer += i;
      }
    System.Console.WriteLine(answer);
  }
}

File: 1/Euler1.vb
Module Euler1
    Public Function multiple_of(ByVal x As Integer, ByVal y As Integer)
        Return (x Mod y) == 0
    End Function
    
    Sub Main()
        Dim i, answer As Integer
        answer = 0
        
        For i = 1 To 999
            If multiple_of(i, 3) Or multiple_of(i, 5) Then answer += i
        Next i
        System.Console.WriteLine(answer)
    End Sub
End Module
    